src/js/angular/security/controllers.js

```diff
@@ -2,6 +2,20 @@ import 'angular/core/services';
 import 'angular/security/services';
 import 'angular/rest/security.rest.service';
 
```

Comment:
I'll probably extract most of these constants in enums to reuse them, because currently such literals are used in many different places.